addons/procurement/procurement.py

```diff
@@ -1,20 +1,18 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-import time
 from psycopg2 import OperationalError
 
-from openerp import api
-from openerp import SUPERUSER_ID
-from openerp.osv import fields, osv
-import openerp.addons.decimal_precision as dp
-from openerp.tools.translate import _
 import openerp
+from openerp import api, fields, models
+from openerp.tools.translate import _
```

Comment:
Now we can directly import translation from odoo.
from odoo import api, fields, models,  _